Lucidtech/Las/Client.cs

```diff
@@ -845,7 +1066,7 @@ public object CreateBatch(string? name = null, string? description = null)
             }
 
             if (startTime != null) {
-                body.Add("startTime", startTime);
+                body.Add("startTime", startTime.Value.ToString("yyyy-MM-ddTHH:mm:ss.ffffzzz"));
```

Comment:
Does ToString("o") work here?

https://docs.microsoft.com/en-us/dotnet/standard/base-types/standard-date-and-time-format-strings

Reply:
Nope, tried it, but it had too many decimals.